src/components/StarRating/StarRating.tsx

```diff
@@ -44,23 +44,20 @@ const TotalReviewer = styled.span`
 export default function StarRating(props: StarRatingProps) {
   const { rating, totalReviewer } = props;
   let label = '별점 정보 없음';
-  if (totalReviewer > 0) {
-    label = `총 리뷰어 ${totalReviewer}명, `
-      + `구매자 평균 별점 ${rating}점`;
+  if (totalReviewer && totalReviewer > 0) {
+    label = `총 리뷰어 ${totalReviewer}명, ` + `구매자 평균 별점 ${rating}점`;
```

Comment:
```suggestion
    label = `총 리뷰어 ${totalReviewer}명, 구매자 평균 별점 ${rating}점`;
```

한 줄로 만들 거라면 아예 이어붙여주세요